greedybear/settings.py

```diff
@@ -13,7 +13,14 @@
 SECRET_KEY = os.environ.get("DJANGO_SECRET", None) or get_random_secret_key()
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = os.environ.get("DEBUG", False) == "True"
+# DEBUG = os.environ.get("DEBUG", False) == "True"
+DEBUG = True
+
+# DB Backup Configs
+DBBACKUP_STORAGE = "django.core.files.storage.FileSystemStorage"
+DBBACKUP_STORAGE_OPTIONS = {"location": "/var/log/greedybear/"}
+# Database Backup Interval (in minutes). default 120 mins = 2 hours.
+DBBACKUP_FREQUENCY = int(os.environ.get("DBBACKUP_FREQUENCY", 120)) == 120
```

Comment:
Not sure if it matters much but instead of `DBBACKUP_STORAGE` you can use `DB_BACKUP_STORAGE` and similarly other variables as well this looks much readable

Reply:
Ah, yes. I was honestly thinking about it while naming it. Sure, will do.

Reply:
Oh, I just remembered. The module depends on those settings [\[ref\]](https://django-dbbackup.readthedocs.io/en/master/installation.html#add-it-in-your-project), so I don't think its possible to rename them (except `DBBACKUP_FREQUENCY`, which I manually added)

Reply:
Ohh i see, that's their default name. It's fine then.

Reply:
Should I change `DBBACKUP_FREQUENCY` or should I keep it as is to represent common configs?

Reply:
I recommend not to change to keep all the DB Backup related vars in a uniform manner.

Reply:
Ok Sir, Got it 👍🏼